### PR TITLE
[VFX] Fix out of experimental

### PR DESCRIPTION
**Pull request by PaulDemeulenaere**
⚠️ ⚠️ ⚠️ Don't merge before C++ PR (88963) ⚠️ ⚠️ ⚠️

### Purpose of this PR
Fix namespace due to the move of out of experimental of the visual effect API.
N.B.: For @sebastienlagarde @JulienIgnace-Unity The change in C++ will grant an internal access of UnityEngine to all code of `Unity.RenderPipelines.HighDefinition.Runtime`. It's temporary, we have to be careful with following PR (don't use internal API without reflection).

---
### Testing status
**Katana Tests**: 🍏 

**Manual Tests**:
- [x] Opened test project + Run graphic tests locally
- [x] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: N/A
Launch Katana :
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?automation-tools_branch=master&ScriptableRenderLoop_branch=vfx%2Ffix%2Fout-of-experimental-namespace&unity_branch=graphics%2Fvfx%2Ffix%2Fout-of-experimental-namespace

Alternative, launch Yamato (Select your branch): Can't be launched because it requires a custom branch

---
### Overall Product Risks
**Technical Risk**: Low
**Halo Effect**: High

---
### Comments to reviewers
We have to be synchronized with the landing of : https://ono.unity3d.com/unity/unity/pull-request/88948/_/graphics/vfx/fix/out-of-experimental-namespace

### CODE REVIEW

File: TestProjects/VisualEffectGraph/Assets/AllTests/Editor/Tests/VFXComponentTest.cs [hunk 1]
```diff
@@ -104,20 +104,20 @@ public void Init()
             camera.transform.LookAt(m_mainCamera.transform);
 
             m_previousCaptureFrameRate = Time.captureFramerate;
-            m_previousFixedTimeStep = UnityEngine.Experimental.VFX.VFXManager.fixedTimeStep;
-            m_previousMaxDeltaTime = UnityEngine.Experimental.VFX.VFXManager.maxDeltaTime;
+            m_previousFixedTimeStep = UnityEngine.VFX.VFXManager.fixedTimeStep;
```
Comment (julienf-unity) [inline TestProjects/VisualEffectGraph/Assets/AllTests/Editor/Tests/VFXComponentTest.cs]: Why is this explicit while the namespace is used? We dont really care though

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs [hunk 2]
```diff
@@ -1006,7 +1007,7 @@ protected override void Render(ScriptableRenderContext renderContext, Camera[] c
                     dynResHandler.SetCurrentCameraRequest(cameraRequestedDynamicRes);
                     RTHandles.SetHardwareDynamicResolutionState(dynResHandler.HardwareDynamicResIsEnabled());
 
-                    UnityEngine.Experimental.VFX.VFXManager.ProcessCamera(camera); //Visual Effect Graph is not yet a required package but calling this method when there isn't any VisualEffect component has no effect (but needed for Camera sorting in Visual Effect Graph context)
+                    VFXManager.ProcessCamera(camera); //Visual Effect Graph is not yet a required package but calling this method when there isn't any VisualEffect component has no effect (but needed for Camera sorting in Visual Effect Graph context)
```
Comment (julienf-unity) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs]: The comment is irrelevant now

Comment (PaulDemeulenaere) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs]: +1 : I think I could update this method invocation in motionVector integration branch

File: TestProjects/VisualEffectGraph/Assets/AllTests/Editor/Tests/VFXComponentTest.cs (hunk 1 above)
Comment (PaulDemeulenaere) [reply TestProjects/VisualEffectGraph/Assets/AllTests/Editor/Tests/VFXComponentTest.cs]: In VFX package, both there are UnityEditor.VFXManager (actually internal) and Engine.VFX.VFXManager are available, in some place, when editor & engine are included, we have to explicit.

**Comment by PaulDemeulenaere** [comment]
The C++ change is in landing pattern, expected to land tonight or tomorrow

**Comment by PaulDemeulenaere** [comment]
C++ has just been merge in trunk.
It should be merge in master as soon as possible !

**Pull request merged**